From 504ad575860a13f180e9d0acae5552dab2f1086e Mon Sep 17 00:00:00 2001
From: Lucas Weaver <l-weaver@ti.com>
Date: Tue, 18 Oct 2022 09:27:51 -0500
Subject: [PATCH] arm64: dts: ti: Adding workaround for existing issue in
 vision apps

-There is an issue with the memory map currently that requires us
 to disable MCU1-0 from the device tree
-This will be fixed cleanly for 8.5, but for now, working around the
 issue by disabling
---
 arch/arm64/boot/dts/ti/k3-j784s4-rtos-memory-map.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/ti/k3-j784s4-rtos-memory-map.dtsi b/arch/arm64/boot/dts/ti/k3-j784s4-rtos-memory-map.dtsi
index caeecc1a0..900a20820 100644
--- a/arch/arm64/boot/dts/ti/k3-j784s4-rtos-memory-map.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-j784s4-rtos-memory-map.dtsi
@@ -119,6 +119,10 @@
 	status = "disabled";
 };
 
+&mcu_r5fss0 {
+	status = "disabled";
+};
+
 &reserved_memory {
 	#address-cells = <2>;
 	#size-cells = <2>;
-- 
2.17.1

